### ISSUE TITLE
[mdb-schema] segfault when using --default-values

### ISSUE THREAD

**Issue by postmaxin** [mdbtools/mdbtools#132]
I am getting a segfault attempting to dump a schema while including default values. It appears to be a null pointer attempting to obtain the property "Format" on a column:

```
	`txtHomePhone`			varchar, 
	`txtHomeCell`			varchar, 
	`txtHomePhone2`			varchar, 
	`txtHomeFax`			varchar, 

Program received signal SIGSEGV, Segmentation fault.
generate_table_schema (export_options=80, dbnamespace=0x0, 
    entry=<optimized out>, outfile=0x7ffff78a7720 <_IO_2_1_stdout_>)
    at backend.c:809
809							if (!strcmp(mdb_col_get_prop(col, "Format"), "Short Date"))
(gdb) p col
$1 = (MdbColumn *) 0x555555875e40
(gdb) p mdb_col_get_prop(col, "Format")
$2 = 0x0
(gdb) 
```

I can naively work around this myself for now but I am not sure that just checking for the null here would be the appropriate choice for all users since I don't really understand the MDB file format..

This is both with the binary that ships with Ubuntu 17.10 and the latest HEAD from github.

**Comment by evanmiller** [comment]
Your solution looks okay to me. It's pretty similar to this section:

https://github.com/brianb/mdbtools/blob/d6f5745d949f37db969d5f424e69b54f0da60b9b/src/libmdb/backend.c#L312-L314

**Comment by evanmiller** [comment]
Fix included in #162, thanks for the contribution!
